scripts/billing/sql/previous_month_costs_by_service_label.sql

```diff
@@ -0,0 +1,21 @@
+/*
+Query returns the previous month's costs and credits by invoice month,
+label value, and service
+
+This example uses a label key of "cost_center" and label value of "sales"
+*/
+
+SELECT
```

Comment:
Please refer to previous_month_costs_by_label.sql for similar pattern.

Reply:
Updated to follow the pattern provided

Reply:
Updated to use similar pattern